resources/views/partials/forms/edit/company-select.blade.php

```diff
@@ -33,7 +33,6 @@
         </div>
         {!! $errors->first($fieldname, '<div class="col-md-8 col-md-offset-3"><span class="alert-msg"><i class="fas fa-times"></i> :message</span></div>') !!}
 
-    {!! $errors->first($fieldname, '<div class="col-md-8 col-md-offset-3"><span class="alert-msg" aria-hidden="true"><i class="fas fa-times" aria-hidden="true"></i> :message</span></div>') !!}
```

Comment:
Wow - completely unexpected catch on this one - should we be carrying the `aria-hidden="true"` attribute over to the "surviving" clauses?

Reply:
I'd love to answer that for you if I could see the change. Safari is 100% covering the deleted line in question. 

<img width="812" alt="Screenshot 2024-07-08 at 5 27 14 PM" src="https://github.com/snipe/snipe-it/assets/197404/7816e255-7ac1-451d-a3a1-1c4844521473">

Reply:
I can see it now - it looked just like a duplicate line, not outside of a for loop or anything. Just the same statement twice. If I'm wrong, I'll happily take edits.

Reply:
You're not wrong - at all - it's just the line you deleted had some `aria-hidden` attributes you might want to move into the line you kept.

Reply:
Totes right on the ` aria-hidden="true"` thing, thank you. Fixing that now.